pkg/spanz/cov.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 PingCAP, Inc.
```

Comment:
what does the file name `cov` mean ?

Reply:
It means convert. Will update the file name.